### PR TITLE
[DDW-1161] Round RAM size in analytics

### PR DESCRIPTION
**Pull request by marcin-mazurek**
This PR addresses the issue of too many RAM size entries with very similar numbers.

Jira: https://input-output.atlassian.net/browse/DDW-1161

### CODE REVIEW

**Review by tomislavhoracek — APPROVED** [review]

**Review by przemyslaw-wlodek — APPROVED** [review]
LGTM 🎉

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,10 +2,15 @@
...
 
+### Features
+
+- Implemented rounding RAM size in analytics ([PR 3088](https://github.com/input-output-hk/daedalus/pull/3088))
+
 ### Chores
 
 - Fixed Storybook state for General stories ([PR 3064](https://github.com/input-output-hk/daedalus/pull/3064))
 
+
```
Comment (tomislavhoracek) [inline CHANGELOG.md:13]: 👌  thanks

**Review by michalrus — APPROVED** [review]
Thank you :bow:

**Review by miorsufianiohk — APPROVED** [review]
LGTM. Great work @marcin-mazurek 👍

**Pull request merged**